kolibri/core/discovery/api.py

```diff
@@ -15,6 +16,10 @@ class NetworkLocationViewSet(viewsets.ModelViewSet):
     permission_classes = [NetworkLocationPermissions | NotProvisionedHasPermission]
     serializer_class = NetworkLocationSerializer
     queryset = NetworkLocation.objects.all()
+    filter_backends = [DjangoFilterBackend]
+    filterset_fields = [
```

Comment:
And I had forgotten that for fields only filtering you can just do this instead of even defining a `FilterSet`!